newton/_src/solvers/vbd/tri_mesh_collision.py

```
@@ -236,6 +239,23 @@ def __init__(
+                                  vertex_triangle_filtering_list,
+                                  vertex_triangle_filtering_list_offsets,
+                                  edge_filtering_list,
+                                  edge_filtering_list_offsets
+                                  ):
+        self.vertex_triangle_filtering_list = vertex_triangle_filtering_list
+        self.vertex_triangle_filtering_list_offsets = vertex_triangle_filtering_list_offsets
+
+        self.edge_filtering_list = edge_filtering_list
+        self.edge_filtering_list_offsets = edge_filtering_list_offsets
+
+    def set_collision_filter_list(self, vertex_triangle_filtering_list, vertex_triangle_filtering_list_offsets, edge_filtering_list, edge_filtering_list_offsets):
```

Ruff (0.14.4)

Redefinition of unused `set_collision_filter_list` from line 242

(F811)